modules/lti/src/main/resources/tools/manage/index.html

```diff
@@ -715,6 +749,52 @@ <h4>Media</h4>
         </li>
         {% } %}
         </ul>
+        </div>
+        <div class="tab-pane" id="captions" style="padding:10px;">
+          <input type="hidden" id="hiddenEvent" name="hiddenEvent"/>
+          <div class="list-group">
+            <div class="list-group-item">
+              <h5 class="list-group-item-heading"><b>Edit Captions</b></h5>
+              <ul class="list-group">
+                 <li class="list-group-item list-group-item-action borderless"><a type="button" href="" id="editCaptions" data-toggle="modal" data-target="#editCaptionsModal"></a></li>
+              </ul>
+            </div>
+            <div class="list-group-item">
+              <h5 class="list-group-item-heading"><b>Download Captions</b></h5>
+              <ul class="list-group">
+                <li class="list-group-item list-group-item-action borderless" style="display:none;" id="downloadNibityCaptions">
+                  <a href="#" class="list-group-item-action" title="Nibity Captions" id="dlNibityCaptions">
+                    <i class="fa fa-download"></i> Way with Words Captions
+                  </a>
+                </li>
+                <li class="list-group-item list-group-item-action borderless" style="display:none;" id="downloadGoogleCaptions">
+                  <a href="#" class="list-group-item-action" title="Google Captions" id="dlGoogleCaptions">
+                    <i class="fa fa-download"></i> Automated Captions
+                  </a>
+                </li>
+                <li class="list-group-item list-group-item-action borderless" style="display:none;" id="downloadUploadedCaptions">
+                  <a href="#" title="Uploaded Captions" id="dlUploadedCaptions">
+                    <i class="fa fa-download"></i> Uploaded Captions
+                  </a>
+                </li>
+              </ul>
+            </div>
+            <div class="list-group-item">
+              <h5 class="list-group-item-heading"><b>Upload New Captions</b></h5>
+              <input autocomplete="off" type="checkbox" id="populatedPresVtt" class="filePopulated" />
+              <span class="fileContainer form-control" data-mediatype="text/vtt" data-title="Choose *.vtt ..." style="display:inline-block;">
+              <input autocomplete="off" type="file" name="text/vtt" id="attachment_captions_webvtt" data-mediatype="vtt" accept=".vtt"/>
+                  <progress min="0" value=0 max="100"></progress>
+              </span>
+              <a href="" id="btnUploadCaptions" class="btn btn-success uploadCaptions" style="display:none;">Upload VTT</a>
+              <textarea style="display:none;" rows="10" cols="50" name="text/vtt"  id="uploadedVttText"></textarea>
+              <span class="errors" style="color:red;display:none;">The file you are trying to upload is not a valid VTT file. 
+                To validate it click <a href="https://quuz.org/webvtt">here</a> and copy and paste the text from your file to the validator. 
```

Comment:
Not a fan of "click here" links. Rather link something semantically meaningful, e.g.

To validate it, use a VTT validator and 

where you link "VTT validator"

Reply:
Noted!